hera_cal/__init__.py

```diff
@@ -34,3 +34,4 @@
 from . import autos
 from . import noise
 from . import tempcal
+from . import nucal
```

Comment:
It's good practice to put a newline at the end of a file

Reply:
Oh, didn't know that, thanks for letting me know. Just added that change.